Makefile

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/make -f
+# SPDX-License-Identifier: MPL-2.0
+#{
+# Copyright 2018-present Samsung Electronics France SAS, and other contributors
```

Comment:
I'm not a lawyer, but I think this repository is copyright Mozilla

Reply:
Ok I guess it's easy to change for me, I tend to reuse existing file when creating new ones